### DISCUSSION TITLE
支付宝支付费用，没有延期

### DISCUSSION THREAD

**Discussion by pinkx7777** [getlantern/lantern-forum-cn#603] [Q&A]
<img width="508" alt="bill" src="https://github.com/getlantern/lantern-forum-cn/assets/173162812/bf489d6b-09fd-4c54-b4c8-123962c4ad98">
账号邮箱地址 pinkx_e@hotmail.com

**Comment by MaxMKMax** [comment]
@pinkx7777 谢谢您支持蓝灯. 好的, 我帮您提交问题. 请您过几天查看这个帖子或邮箱. 我们会给您发激活码.
